### ISSUE TITLE
Bug detected in loadDatabases

### ISSUE THREAD

**Issue by Yu-sysbio** [SysBioChalmers/GECKO#184]
### Description of the bug:
Not able to retrieve data from UniProt via API

#### Expected behavior:
Download UniProt data

#### Current behavior:
Error using matlab.internal.webservices.HTTPConnector/copyContentToByteArray (line 396)
The server returned the status 500 with message "Internal Server Error" in response to the request to URL
https://rest.uniprot.org/uniprotkb/query=taxonomy:9606&format=tab&compress=no&columns=id,genes(OLN),genes(PREFERRED),ec,database(GeneID),database(RefSeq),mass,sequence.

Error in readContentFromWebService (line 46)
        byteArray = copyContentToByteArray(connection);

Error in webread (line 125)
[varargout{1:nargout}] = readContentFromWebService(connection, options);

#### Reproducing these results:

```matlab
% If you worked on matlab, paste your code here
```
taxonID = '9606';
url = ['https://www.uniprot.org/uniprot/?query=taxonomy:' taxonID ...
        '&format=tab&compress=no&columns=id,genes(OLN),genes(PREFERRED),ec,database(GeneID),database(RefSeq),mass,sequence'];
uniprotDL = webread(url);

```python
# If you worked on python, paste your code here
```

#### Context

<details>
* Operating System: macOS version 12.6.1
* Python version:
* Matlab version: 2019b
</details>

**I hereby confirm that I have:**
- [x] Tested my code with [all requirements](https://github.com/SysBioChalmers/GECKO) for running GECKO
- [ ] Done this analysis in the `master` branch of the repository
- [x] Checked that a similar issue does not exist already

**Comment by edkerk** [comment]
UniProt has indeed changed their API, @johan-gson solved this in #180, has not been reviewed yet.

**Comment by ae-tafur** [comment]
urlread function works for uniprot
